airflow/providers/openlineage/plugins/facets.py

```diff
@@ -39,15 +39,31 @@ class AirflowMappedTaskRunFacet(BaseFacet):
 
     @classmethod
     def from_task_instance(cls, task_instance):
-        task = task_instance.task
-        from airflow.providers.openlineage.utils.utils import get_operator_class
+        from airflow.providers.openlineage.utils.utils import get_fully_qualified_class_name
 
         return cls(
             mapIndex=task_instance.map_index,
-            operatorClass=f"{get_operator_class(task).__module__}.{get_operator_class(task).__name__}",
+            operatorClass=get_fully_qualified_class_name(task_instance.task),
         )
 
 
+@define(slots=False)
+class AirflowJobFacet(BaseFacet):
+    """Composite Airflow job facet."""
+
+    taskTree: dict
+    taskGroups: dict
+    tasks: dict
+
+
+@define(slots=False)
+class AirflowStateRunFacet(BaseFacet):
+    """Airflow facet providing state information."""
+
+    dagRunState: str
+    tasksState: dict[str, str]
```

Comment:
Maybe include that information into `AirflowRunFacet`?

Reply:
Ah, makes sense. I would change the description of those facets though to indicate more where they are used and how.

Reply:
Exactly. We need that information in the dag-level event. Added description to both facets, so it's easier to deduct how to use them and what they contain.